src/core/sagas/search.js

```diff
@@ -0,0 +1,41 @@
+// Disabled because of
+// https://github.com/benmosher/eslint-plugin-import/issues/793
+/* eslint-disable import/order */
+import { call, put, select, takeLatest } from 'redux-saga/effects';
+/* eslint-enable import/order */
+
+import { searchLoad } from 'core/actions/search';
+import { search as searchApi } from 'core/api';
+import { SEARCH_STARTED } from 'core/constants';
+import log from 'core/logger';
+import { createErrorHandler, getState } from 'core/sagas/utils';
+
+
+export function* fetchSearchResults({ payload }) {
+  const { errorHandlerId } = payload;
+  const errorHandler = createErrorHandler(errorHandlerId);
+
+  yield put(errorHandler.createClearingAction());
+
+  try {
+    const { filters, page } = payload;
+
+    const state = yield select(getState);
+
+    const response = yield call(searchApi, {
+      api: state.api,
+      auth: state.auth,
+      filters,
+      page,
+    });
+
+    yield put(searchLoad({ page, filters, ...response }));
```

Comment:
Oops, I just noticed the `...response` spread. Can you change this to explicit parameters? I know this is just copied from the old code but we've had too much trouble with spreads. 

Also, `searchLoad()` doesn't take a `page` argument at all. Flow helps to catch these types of problems but until you're using it, please double check all function signatures manually.